sqlmesh/core/model/kind.py

```diff
@@ -59,7 +65,12 @@ def is_materialized(self) -> bool:
     @property
     def only_latest(self) -> bool:
         """Whether or not this model only cares about latest date to render."""
-        return self.model_kind_name in (ModelKindName.VIEW, ModelKindName.FULL)
+        return self.is_view or self.is_full
+
+    @property
+    def is_time_based_load(self) -> bool:
+        """Whether or not this model has a time column which influences the load pattern used."""
+        return self.is_incremental_by_time_range
```

Comment:
Do we expect other combinations of flags to fall into the category of "is_time_based_load"? If not, do we need this?

Reply:
i think this is removed already

Reply:
what do you mean? Am I looking at the wrong PR?

Reply:
This PR is based off of my other PR. So some fixes are in my other one that haven't been merged into this one yet. Will be updating soon.